opentreemap/importer/views.py

```diff
@@ -110,6 +113,18 @@ def get_import_table(request, instance, table_name):
 _EVENT_TABLE_PAGE_SIZE = 5
 
 
+def _cleanup_tables(instance):
+    ievents = (list(TreeImportEvent.objects.filter(instance=instance)) +
+               list(SpeciesImportEvent.objects.filter(instance=instance)))
+    for ie in ievents:
+        if ie.task_id != '':
+            result = AsyncResult(ie.task_id)
+            if result.failed():
```

Comment:
I think we should mark it as failed if there's no `AsyncResult` *and* the job is in a "running" state. Reasons:
* It will allow LA to restart their stalled imports
* Seems like if the celery task is actually running then `AsyncResult(ie.task_id)` should always work.